src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_PROPERTY_VALUE = 100;
+    private static final int NUMBER_OF_FIGURES = 5;
+    private String color = new ColorSupplier().getRandomColor();
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(NUMBER_OF_FIGURES);
+        switch (figureNumber) {
+            case 1:
+                return new IsoscelesTrapezoid(color,random.nextInt(MAX_PROPERTY_VALUE),

```

Comment:
```suggestion
                return new IsoscelesTrapezoid(color, random.nextInt(MAX_PROPERTY_VALUE),

```
add spaces in other places like this